src/_layouts/redirect.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en-US">
```

Comment:
Shall we keep this as `en-GB` in line with `default.html`?

Reply:
Yeah that makes sense. It's in our codebase now should so match the standards. I'll update this template based on the default.html